subsys/net/ip/tcp2.c

```diff
@@ -1117,6 +1120,79 @@ static void tcp_fin_timeout(struct k_work *work)
 	net_context_unref(conn->context);
 }
 
+#if IS_ENABLED(CONFIG_NET_TCP_KEEPALIVE)
+static void tcp_send_keepalive_probe(struct k_work *work)
+{
+	struct tcp *conn = CONTAINER_OF(work, struct tcp, keepalive_timer);
+
+	struct net_pkt *pkt;
+	int ret = 0;
+
+	if(!(net_context_get_state(conn->context) & NET_CONTEXT_CONNECTED))
+	{
```

Comment:
Please put the starting { previous line, please check rest of the PR for this issue
Also there needs to be a space between `if` and `(`